docs/operations/security-overview.md

```diff
@@ -264,3 +318,29 @@ As an alternative to using the basic metadata authenticator, as shown in the pre
 
 
 Congratulations, you have configured permissions for user-assigned roles in Druid!
+
+
+## Druid security trust model
+Like all other security systems, trust is the foundation of the Druid security model. Druid administrators and read-only users are trusted users. Therefore, they are not expected to act maliciously.
+
+
+Based on this expectation, Druid operates according to the following principles:
+
+From the inner most layer:
+1. Druid processes run within the system user context. They have access to the local files granted to the specified system user.
+2. The Druid ingestion system can create new processes to execute tasks. Those tasks inherit the user of their parent process. This means that any user authorized to submit an ingestion task can use the ingestion task permissions to read or write any local files that the Druid process has access to.
+
+> Note: Only grant the permission to submit ingestion tasks to trusted users because they can read and write to local file system.
+
+Within the cluster:
+1. Druid assumes it operates on an isolated, protected network where no reachable IP within the network is under adversary control. It is the responsibility of system implementers to setup firewalls and other methods of protection.
+2. Druid supports TLS encryption for network traffic, including API calls and data transfers.
```

Comment:
I don't think TLS encryption is part of the trust model.

Reply:
Any suggestions on how we describe that the network traffic is protected?

Reply:
@2bethere , @suneet-s TLS is "A" trust model if not "Druid's" model. We have already mentioned in Best Practices section to Enable TLS. So maybe we are covered on this front?

Reply:
Ha, I see. Maybe this:
```suggestion
2. Druid assuming network traffic within the cluster is encrypted, including API calls and data transfers. By default, this is implemented via TLS.
```

Reply:
```suggestion
Druid assumes network traffic within the cluster is encrypted, including API calls and data transfers. The default encryption implementation uses TLS.
```